examples/clock/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::spawn_local;
+use gloo::timers::future::IntervalStream;
+use futures_util::{stream::StreamExt, future::ready};
+use chrono::Timelike;
+
+#[wasm_bindgen(start)]
+pub fn main() {
+    console_error_panic_hook::set_once();
+    let document = web_sys::window().unwrap_throw().document().unwrap_throw();
```

Comment:
Nit: could you add some empty lines in here to spread things out so they look nicer?

Reply:
I've tried to group things together in a sensible way, could you have a look and tell me what you think.

Reply:
I'd personally prefer a newline between every statement, but I guess it doesn't matter too much.

Reply:
@Pauan I've added newlines - is this what you mean?

Reply:
Looks great, thanks!